app/src/main/java/org/wikipedia/bridge/JavaScriptActionHandler.kt

```diff
@@ -39,16 +40,19 @@ object JavaScriptActionHandler {
     @JvmStatic
     fun setUp(): String {
         val app: WikipediaApp = WikipediaApp.getInstance()
+        val topActionBarHeight = (Math.round(app.getResources().getDimensionPixelSize(R.dimen.lead_no_image_top_offset_dp) / getDensityScalar()))
```

Comment:
Looks like it can be simplified to:
`(app.resources.getDimensionPixelSize(R.dimen.lead_no_image_top_offset_dp) / getDensityScalar()).roundToInt()`